v8py/jsfunction.cpp

```diff
@@ -35,12 +36,24 @@ PyObject *js_function_call(js_function *self, PyObject *args, PyObject *kwargs)
         js_this = self->js_this.Get(isolate);
     }
     int argc = PyTuple_GET_SIZE(args);
-    Local<Value> *argv = new Local<Value>[argc];
-    jss_from_pys(args, argv, context);
+    MaybeLocal<Value> result;
 
     if (!context_setup_timeout(context)) return NULL;
-    MaybeLocal<Value> result = object->CallAsFunction(context, js_this, argc, argv);
-    delete[] argv;
+#ifndef _WIN32
+    // error C2131: expression did not evaluate to a constant on Windows
+    if (argc <= 16) {
+        std::vector< Local<Value> > argv(argc);
```

Comment:
`std::vector` allocates on the heap, doesn't it? If so there's no performance benefit over a straight heap-allocated array, and therefore no need for this branch and #ifdef.